backend/danswer/server/models.py

```diff
@@ -27,6 +27,11 @@ class DataRequest(BaseModel):
     data: str
 
 
+class HelperResponse(BaseModel):
```

Comment:
In case we have other Helper models or the intent model can do aggregations and needs to be called further down the line etc. Hence the more freeform return